phat/contracts/lego_rs/Cargo.toml

```diff
@@ -0,0 +1,38 @@
+[package]
+name = "lego_rs"
+version = "0.1.0"
+authors = ["[your_name] <[your_email]>"]
+edition = "2021"
+
+[dependencies]
+ink = { version = "4.2.0", default-features = false }
+pink = { package = "pink-extension", version = "0.4", default-features = false }
+
+scale = { package = "parity-scale-codec", version = "3", default-features = false, features = ["derive"] }
+scale-info = { version = "2.6", default-features = false, features = ["derive"], optional = true }
+serde = { version = "=1.0.185", default-features = false, features = ["derive"] }
+pink-json = { version = "0.4.0", default-features = false }
+hex = { version = "0.4", default-features = false, features = ["alloc"] }
+this-crate = "0.1"
+
+[dev-dependencies]
+ink_e2e = "4.2.0"
+
+[lib]
+path = "lib.rs"
+
+[features]
+default = ["std"]
+std = [
+    "ink/std",
+    "pink/std",
+    "scale/std",
+    "scale-info/std",
+    "pink-json/std",
+    "serde/std",
+]
+ink-as-dependency = []
+e2e-tests = []
+
+[patch.crates-io]
```

Comment:
Do we need patches here to support big JS input?

Reply:
This PR is only a workaround for the performance issue of the JS lego.It just implements the only used command `call` without args in the bricks. So, looks like it doesn't need big IO buffer.